### PR TITLE
Added ise-web-security-bundle flex recipe

### PR DESCRIPTION
**Pull request by henrym2**
| Q             | A
| ------------- | ---
| License       | MIT

### CODE REVIEW

File: googlechromelabs/ise-web-security-bundle/1.0/config/packages/ise_web_security.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+                        active: true
+                        policy: 'same-origin'
+                coep: 
+                        active: true
+                        policy: 'require-corp'
+                fetch_metadata:
+                        active: true
+                        allowed_endpoints: []
+                trusted_types:
+                        enabled: false
```
Comment (jbtcd) [inline googlechromelabs/ise-web-security-bundle/1.0/config/packages/ise_web_security.yaml:13]: It looks to me like your indentation is always 8 spaces, but if I understand the [requirements](https://github.com/symfony/recipes/blob/master/README.rst#validation) correctly it should only be 4.

> YAML files must use 4 space indentations;

>

Comment (henrym2) [reply googlechromelabs/ise-web-security-bundle/1.0/config/packages/ise_web_security.yaml]: Good catch! Had vs-code's tab-to-spaces setup wrong on accident. Thanks!

File: googlechromelabs/ise-web-security-bundle/1.0/manifest.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
+{
+	"bundles": {
```
Comment (jbtcd) [inline googlechromelabs/ise-web-security-bundle/1.0/manifest.json:2]: Oh, similar mistake as in the yaml file, here you mixed 2 and 4 spaces, but should be 4 everywhere :thinking: 
Didn't see that just now, sorry for the double work :see_no_evil:

**Review by Nyholm — APPROVED** [review]
Thank you

**Pull request merged**